packages/app/src/systems/Bridge/components/BridgeTxItem/BridgeTxItem.tsx

```diff
@@ -7,10 +7,10 @@ type BridgeTxItemProps = {
   date?: Date;
   fromLogo: ReactNode;
   toLogo: ReactNode;
-  asset: {
-    assetImageSrc: ReactNode | string;
-    assetAmount: string;
-    assetSymbol: string;
+  asset?: {
+    assetImageSrc?: ReactNode | string;
+    assetAmount?: string;
+    assetSymbol?: string;
```

Comment:
Why we asset become optional right now?

Reply:
it's just to improve flexibility and avoid needing conditionals in other place, as the asset can be undefined and will be set only after getting the tx info